react/features/deep-linking/components/DeepLinkingMobilePage.web.js

```diff
@@ -139,6 +148,15 @@ class DeepLinkingMobilePage extends Component<Props> {
                         { t(`${_TNS}.openApp`) }
                         {/* </button> */}
                     </a>
+                    {
+                        isSupportedMobileBrowser()
+                            && <a
```

Comment:
addd target blank for external API users?

Reply:
my bad, how could I forget that :)